packages/config-loader/src/sources/transform/include.ts

```diff
@@ -48,7 +50,9 @@ export function createIncludeTransform(
     }
     // Check if there's any key that starts with a '$', in that case we treat
     // this entire object as an include description.
-    const [includeKey] = Object.keys(input).filter(key => key.startsWith('$'));
+    const [includeKey] = Object.keys(input).filter(key =>
+      INCLUDE_KEYS.includes(key),
+    );
```

Comment:
This is a tricky one that could use some discussion. In config we currently use `$` as a prefix for keys that should be processed by the config loader. This fixes it but mixing different usages of `$` prefixes still has the risk to cause confusion. One option could be to choose a different prefix for the predicates, or choose a completely different structure that doesn't require prefixing.